nbgrader/html/formgrade.py

```diff
@@ -148,6 +148,9 @@ def view_submission(submission_id):
     if not os.path.exists(filename):
         abort(404)
 
+    if not app.notebook_server_exists:
+        abort(503, 'Submission viewing notebook server not started.')
+
```

Comment:
@jhamrick one step further, I could modify the templates to not `a href` the `/submissions/#` url (just the formgrade.tpl?) so the grader wouldn't have to see the 503.  I can do that in this/another PR if you'd like.